tool/tctl/common/resource_command.go

```diff
@@ -856,34 +935,24 @@ func (rc *ResourceCommand) createDevice(ctx context.Context, client auth.ClientI
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	name := dev.Id
+	_, err = client.DevicesClient().GetDevice(ctx, &devicepb.GetDeviceRequest{DeviceId: dev.GetId()})
```

Comment:
Using multiple means for achieving the same end is unnecessary complexity
```suggestion
	_, err = client.DevicesClient().GetDevice(ctx, &devicepb.GetDeviceRequest{DeviceId: name})
```